ddp-workspace/projects/ddp-singular/src/app/config/feature-flags/feature-flags.enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum FeatureFlagsEnum {
+  DDP_8404_Home_page_update = 'feature DDP_8404_Home_page_update',
+  DDP_8560_Dashboard_page_update = 'feature DDP_8560_Dashboard_page_update'
+}
```

Comment:
Would suggest renaming to make it clearer what true or false flag would mean, e.g.: 
**Show**DDP8404HomePageUpdate

Reply:
Renamed.